### PR TITLE
Fix/1 jopa constraint violation

### PR DESCRIPTION
**Pull request by grigobog**

### CODE REVIEW

**Comment by blcham** [comment]
@grigobog I believe that you said, I should review the changes here. Please, next time add me as Reviewer, which indicates to me that i should review the pull request.

**Review by blcham — COMMENTED** [review]
see my comments.

File: bpmn2stamp-converter/src/main/java/cz/cvut/kbss/bpmn2stamp/converter/persistance/RdfRepositoryWriter.java [hunk 1]
```diff
@@ -56,7 +58,27 @@ public <T> void write(Iterable<T> thingsToWrite) {
...
+            if (re.getCause() == null) {
+                System.err.println("Unknown cause.");
+                return;
+            }
+            
+            if (re.getCause() instanceof CardinalityConstraintViolatedException) {
+                System.err.println("Constraints violation error: " + re.getCause().getMessage());
+            } else {
+                System.err.println("Unknown error.");
+                re.getCause().printStackTrace();
```
[the remark this answers is not in the document]
Comment (grigobog) [reply bpmn2stamp-converter/src/main/java/cz/cvut/kbss/bpmn2stamp/converter/persistance/RdfRepositoryWriter.java:76]: already fixed in https://github.com/kbss-cvut/bpmn2stamp/issues/24

File: bpmn2stamp-converter/src/test/java/cz/cvut/kbss/bpmn2stamp/service/data/jednani-sag-organization-structure.ttl [hunk 2]
```diff
@@ -53,99 +56,76 @@
...
-    <http://BPMNbasedOntology#is-role-in> <http://onto.fel.cvut.cz/ontologies/ucl/example/jednani-sag-organization-structure/sekce>;
-    rdfs:label "Ředitel sekce" .
-  
-  <http://onto.fel.cvut.cz/ontologies/ucl/example/jednani-sag-organization-structure/ředitel_úcl>
-    a owl:NamedIndividual, <http://BPMNbasedOntology#Role>;
-    rdfs:label "Ředitel ÚCL" .
-}
+# Generated by the OWL API (version 5.1.19) https://github.com/owlcs/owlapi/
+
+<../jednani-sag-organization-structure> a owl:Ontology;
```
[the remark this answers is not in the document]
Comment (grigobog) [reply bpmn2stamp-converter/src/test/java/cz/cvut/kbss/bpmn2stamp/service/data/jednani-sag-organization-structure.ttl:61]: Agreed to close for now

**Pull request merged**